src/plcc/load_spec/validate_spec/validate_syntactic_spec/Grammar.py

```diff
@@ -0,0 +1,56 @@
+from .errors import InvalidNonterminalError, InvalidFormError
```

Comment:
Please create a subdirectory for checkLL1, and move everything related to it into that subdirectory.